DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan/test_vlan_configuration_checks.py

```diff
@@ -0,0 +1,83 @@
+import json
+import pytest
+
+from dent_os_testbed.lib.bridge.bridge_vlan import BridgeVlan
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from dent_os_testbed.utils.test_utils.tb_utils import tb_get_all_devices
+
+pytestmark = pytest.mark.suite_functional_vlan
+
+
+@pytest.mark.asyncio
+async def test_can_set_max_vlans(testbed):
+    """
+    Test Name: Maximum vlans for the interface
+    Test Suite: suite_functional_vlan
+    Test Overview: Test maximum number of vlans that can be set on interface
+    Test Procedure:
+    1. Initiate test params
+    2. Create bridge entity and set state to "up" state.
+    3. Enslave interface to the created bridge entity.
+    4. Insert interface to all VLANs possible (4094)
+    5. Verify interface is in all possible (4094) VLANs
+    """
+
+    max_vlans = 4094
+    #  1.Initiate test params
+    dent_devices = await tb_get_all_devices(testbed)
+    device = dent_devices[0].host_name
+    test_port = dent_devices.links_dict[device][1][0]  # get first port from config
```

Comment:
In fact, you might not have any port in config. It should be enough just to use "swpX"
But let's leave as is so far.